### PR TITLE
Remove symfony/finder dependency

### PR DESCRIPTION
**Pull request by rdss-sknott**

### CODE REVIEW

File: infection.json.dist [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "timeout": 10,
-    "minMsi": 84,
-    "minCoveredMsi": 84,
+    "minMsi": 86,
+    "minCoveredMsi": 86,
```
Comment (rdss-pwacht) [inline infection.json.dist:4]: ❤️ ^^das kann süchtig machen!

**Review by rdss-pwacht — APPROVED** [review]

**Review by rdss-pwacht — DISMISSED** [review]
Dependency-problem on one shop with zooroyal/platformtestingbasics 0.2 -> maybe time to change this!

**Comment by rdss-sknott** [comment]
> Dependency-problem on one shop with zooroyal/platformtestingbasics 0.2 -> maybe time to change this!

Dependency was runter geschraubt. sollte jetzt klappen ^^

File: src/main/php/CommandLine/Factories/Exclusion/StaticExcluder.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+namespace Zooroyal\CodingStandard\CommandLine\Factories\Exclusion;
+
+use Zooroyal\CodingStandard\CommandLine\Library\Environment;
+
+class StaticExcluder implements ExcluderInterface
+{
+    private Environment $environment;
+    /** @var array<string>  */
+    private array $pathsToExclude = [
```
Comment (rdss-pwacht) [inline src/main/php/CommandLine/Factories/Exclusion/StaticExcluder.php:11]: Maybe a thing for config-file in next improvements?

**Review by rdss-pwacht — APPROVED** [review]

**Review by rdss-neljallouli — APPROVED** [review]

**Pull request merged**